connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -192,6 +192,12 @@ class SparkSession(
     new Dataset[T](this, plan)
   }
 
+  private[sql] def newCommand[T](f: proto.Command.Builder => Unit): proto.Command = {
```

Comment:
Are we going to use this method else where? Otherwise it seems a bit much to add.

Reply:
I think for any other command related API, we can use it.